[WIP] Inclusion of Pluggable Components (.NET) SDK docs (#3088)
dapr/docs: daprdocs/content/en/developing-applications/develop-components/pluggable-components/pluggable-components-sdks/_index.md

---
type: docs
title: "Pluggable components SDKs"
linkTitle: "SDKs"
weight: 2000
description: "Develop pluggable components in your favorite language"
no_list: true
is_preview: true
---

> The Dapr SDKs are the easiest way for you to create Dapr Pluggable Components. Choose your favorite language and get up and running with Dapr in minutes.

Reviewer: ```suggestion
The Dapr SDKs are the easiest way for you to create pluggable components. Choose your favorite language and start creating components in minutes.
```

Author: Generally agree, but I find "get up and running creating components" a bit awkward still.  How about:

*The Dapr SDKs are the easiest way for you to create pluggable components. Choose your favorite language and start creating components in minutes.*

> ## Pluggable Components SDKs

Reviewer: ```suggestion
## Pluggable components SDKs
```

| Language | Status |
|----------|:------:|

> | [.NET]({{< ref pluggable-components-dotnet >}}) | In development |

Reviewer 2: ```suggestion
| [.NET]({{< ref dotnet-sdk-docs >}}) | In development |
```

> | [Java]({{< ref java >}}) | In development |

Author: As the docs for Go and Java aren't available yet, should these rows be removed?

Author: Removed.
